PHPStan e CodeSniffer

andrelugomes opened the pull request:
Uma prévia sem os Logs

O Outro PR tem os Logs

diniz-victor commented on src/Producer.php:
```diff
@@ -0,0 +1,36 @@
+class Producer
+{
+    private Configuration $config;
+    private LoggerInterface $logger;
```
Não falta o ? para LoggerInterface?

diniz-victor reviewed (commented):
Sugiro revisar a necessidade de comentários nos métodos das classes (tem até um TODO PHP DOC).

andrelugomes commented:
> Sugiro revisar a necessidade de comentários nos métodos das classes (tem até um TODO PHP DOC).

Seguindo alguma documentação principalmente em métodos públicos @diniz-victor .

Qual seria a sua ideia?

Commit:
Ajustes para atender PHPStan e CodeSniffer

Para que o projeto inicial, devemos seguir preticas e especificações.

Adicionamos os checkers e fizemos diversos ajustes.

Commit:
fixup! Ajustes para atender PHPStan e CodeSniffer

ruschoni02 commented on examples/producers/simple_producer.php:
```diff
@@ -1,13 +1,9 @@
-use PHP\Kafka\Exceptions\KafkaConsumerException;
+use PHP\Kafka\Config\ProducerConfiguration;
 
 require '../../vendor/autoload.php';
```
Os import antes do require num da ruim não?

andrelugomes replied:
uai.. vou confirmar

ruschoni02 reviewed (approved):
Faz sentido passar a mensagem para as exception, não seria uma unica?
